packages/types/types/test.tsx

```diff
@@ -37,6 +42,16 @@ borders({
 
 hasPaletteColor({ theme }) // $ExpectType boolean
 
+color({ theme, bg: 'primary', color: 'secondary' }) // $ExpectType string | string[]
+
+getPaletteColor('primary', 'light') // $ExpectType (props: ThemeArgs) => any
+
+getTextColorOn('bla') // $ExpectType (props: ThemeArgs) => any
+
+createTheme(theme, {}) // $ExpectType CreateThemeResult
+
+const myTheme: ThemeType = theme
```

Comment:
Is `myTheme` needed still?

Reply:
I think so since this just asserts that `typeof theme` is compatible with `ThemeType`

Reply:
Ahh I see, just wanted to make sure!